src/api_server/src/lib.rs

```diff
@@ -295,7 +294,6 @@ mod tests {
                 snapshot_type: SnapshotType::Diff,
                 snapshot_path: PathBuf::new(),
                 mem_file_path: PathBuf::new(),
-                version: None,
```

Comment:
Shouldn't we also remove `ser_fn` functions like [this](https://github.com/firecracker-microvm/firecracker/blob/9b0e03a437c753d40c29647e63f35f4fdb249fce/src/vmm/src/vstate/vcpu/aarch64.rs#L295) from versionize? They are only used for creation of old versions of snaps.

Reply:
You are right! I will grep to find a list of these and remove.

Reply:
done!